editor/plugins/tiles/tile_map_editor.cpp

```diff
@@ -124,6 +124,7 @@ void TileMapEditorTilesPlugin::_tab_changed() {
 void TileMapEditorTilesPlugin::_update_tile_set_sources_list() {
 	// Update the sources.
 	int old_current = sources_list->get_current();
+	int old_source = sources_list->get_item_metadata(old_current);
```

Comment:
I am not sure it can happen, but this might trigger an error if `old_current` is negative (there is no source selected I guess). There are several conditions checking for `old_current >= 0` later on.